.env.example

```diff
@@ -42,6 +42,15 @@ MAIL_FROM_NAME=
 MAILGUN_DOMAIN=
 MAILGUN_SECRET=
 
+# Slack Webhook Endpoint path to be appended to https://hooks.slack.com/services/.
+SLACK_ENDPOINT=
+
+# Default slack channel for the bot to post in.
+SLACK_CHANNEL=
+
+# Default username to post as
+SLACK_USERNAME=
```

Comment:
All you need to do to update your pull request is change the code, commit it, and then push to your branch. In this case, your local development branch. You will see at the bottom of the pull request, it will show the new commits.